pegasus/sites.v3/hourofcode.com/i18n/en.yml

```
@@ -2579,7 +2579,7 @@
   tutorial_codesterscovid_longdescription: "Learn the basics of coding in Python while creating your own PSA about how we can stop the spread of COVID-19. Design your own animated interactive PSA to share critical information about the importance of social distancing, wearing a mask, quarantining, and other measures."
   tutorial_ttbunker_longdescription: "You found The Bunker, and all its secrets will be yours, once you crack its encryption. An army approaches, and you must hold the perimeter using the power of programming. Code individual towers, setting range, damage, and more special powers by adding new lines of code and changing values. Survive long enough to learn how classes and instances can make you even more powerful."
   tutorial_cuttle_longdescription: "Challenge yourself with these fun and interactive tasks and learn all about computational thinking."
-  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! "
+  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! #codeisyourvoice"
```

Adding missing spaces
```suggestion
  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! #codeisyourvoice"
```